lib/icalendar/alarm.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ICalendar.Alarm do
+  @moduledoc """
+  Events have alarms.
+  """
+
+  defstruct minutes_before: 60
```

Comment:
I took some days off...
Sounds like a good idea.

What about supporting the erlang version for now and adding the new elixir
time type as soon as it was released?

Louis Pilfold <notifications@github.com> schrieb am Du. 18. Ouj. 2016 um
22:37:

> In lib/icalendar/alarm.ex
> <https://github.com/lpil/icalendar/pull/4#discussion_r75384499>:
>
> > @@ -0,0 +1,21 @@
> > +defmodule ICalendar.Alarm do
> > +  @moduledoc """
> > +  Events have alarms.
> > +  """
> > +
> > +  defstruct minutes_before: 60
>
> Hi @elitau <https://github.com/elitau> , what do you think?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/lpil/icalendar/pull/4/files/f7c822bf065c4ecf2ceead068b3fa40e1ed24529#r75384499>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABKpjpRx1E3t7TOPRs9rK5SMqxxSNP9ks5qhMKmgaJpZM4JjynG>
> .
>